Tests/DCoreKitTests/MessagingApiTests.swift

```diff
@@ -86,6 +90,6 @@ class MessagingApiTests: XCTestCase {
         ("testGetAllResponsesByReceiverUsingWss", testGetAllResponsesByReceiverUsingWss),
         ("testGetAllResponsesBySenderUsingWss", testGetAllResponsesBySenderUsingWss),
         ("testGetAllDecryptedMessagesBySenderUsingWss", testGetAllDecryptedMessagesBySenderUsingWss),
-        ("testGetAllDecryptedMessagesBySenderUsingWrongCredentialsUsingWss", testGetAllDecryptedMessagesBySenderUsingWrongCredentialsUsingWss),
+        ("testZLastGetAllDecryptedMessagesBySenderUsingWrongCredentialsUsingWss", testZLastGetAllDecryptedMessagesBySenderUsingWrongCredentialsUsingWss),
```

Comment:
typo ZLast

Reply:
it's not a typo, it's intentional. Tests are run based on alphabetical ordering, so I wanted to make sure that this test will run last

Reply:
Order should not matter? what if test running behaviour changes from alpha-numeric to fully random? I understand u need some data before hand, can u add those data into test?

Reply:
all tests in for messaging api have been modified to work regardless of order of execution